test/unit/snappyex_test.exs

```diff
@@ -0,0 +1,81 @@
+defmodule Unit.SnappyExTest do
+  use ExUnit.Case
+  use ExUnitProperties
+  alias SnappyEx
+  doctest SnappyEx
+
+  def stream_identifier, do: <<0xFF, 0x06, 0x00, 0x00, 0x73, 0x4E, 0x61, 0x50, 0x70, 0x59>>
+
+  def assert_snappy_decompress_frames(compressed, uncompressed) do
+    {:ok, ^uncompressed} =
+      compressed
+      |> Base.decode16!()
+      |> SnappyEx.decompress_frames()
+  end
+
+  test "test valid stream identifier" do
```

Comment:
We should have three separate tests, one per assert here, so we can run them separately and they can fail separately.